docs/contributing/README.md

```diff
@@ -14,20 +14,22 @@ You can also reach out for support in the `#sig-spyre` channel in the [vLLM Slac
 
 ## Developing
 
-### Building the docs with MkDocs
+### Docs
```

Comment:
Still undecided whether docs should be its own heading or should be within `developing`. We don't really have anything else under `developing`, which is kind of confusing.